src/utils/section-handler.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+function section() {
+  return function handler(h, node, _, handlechild) {
+    const sectionnode = h(node, 'div', { class: 'section' });
```

Comment:
Maybe `sectioNode`?

Also, should we prefix the class to avoid potential collisions with other frameworks or the customer classes?

What about following https://suitcss.github.io/ and use something like `hlx-section`